src/components/accessibility-link.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import {IssueOpenedIcon} from '@primer/octicons-react'
+
+export function AccessibilityLink({label}){
+    return <><IssueOpenedIcon /> <a target="_blank" href={`https://github.com/github/primer/issues?q=is%3Aopen+is%3Aissue+label%3A%22component%3A+${label}%22+label%3Aaccessibility`}> View open accessibility issues related to this component</a></>
+}
```

Comment:
Can we use `Link` from PRC here? styling looks a bit different compared to other links

<img width="616" alt="Screenshot 2023-09-07 at 08 25 02" src="https://github.com/primer/design/assets/912236/6c9094ed-a4c3-4c41-917d-49c894bb7f7e">


Also, I think that using target="_blank" is not encouraged from accessibility. More in this thread:

https://github.slack.com/archives/C03RM57QL64/p1688042180943279